packages/cardano-services/src/Program/options/postgres.ts

```diff
@@ -32,66 +32,122 @@ export interface PosgresProgramOptions {
   postgresSslCaFile?: string;
 }
 
-export const withPostgresOptions = (command: Command) =>
-  command
+export type ConnectionNames = '' | 'Projection';
```

Comment:
How about `ConnectionNames = 'DbSync' | 'Projection'` rather than`'' | 'Projection'` ?

Reply:
The main reason is to make the lesser impact as possible.

Your proposal would require to change all the occurrences of `POSTGRES_DB` (and all other environment variables and command line option) in `POSTGRES_DB_DBSYNC`.

Reply:
Ah right, well that's an option to accept no suffix is equal to db-sync. Fair enough based on the impact otherwise.

Reply:
> well that's an option to accept "no suffix" is equal to "db-sync"

Exactly

Reply:
Both reasonable points. I'll throw another one in the mix :laughing: 

`ConnectionNames = '' | 'DbSync';`

Why? Using the name "Projection" is a bit jarring to me, and it's not that specific since db-sync is also a projection. I think we can assign the non-specific variant to the database defined within the SDK. Yes it's a refactor now, but it will be less impactful when we fully replace db-sync, since we can just drop the config to be left with what we have now.

Reply:
Yes, breaking change is inevitable if we think that a future version will only need a single pg connection, so best to do it now while we're sub-1.0 :sunglasses: @mkazlauskas thoughts?

Reply:
This last consideration was enough to convince me: I'll apply this change

Reply:
I agree it's best to do it now. I would also rename the type to `ConnectionName`

Reply:
fae7aaff and subsequent fixes should solve this

Reply:
cba27e3cf should finalize the change requested her